Solution-Week1/script_1_check_primenum.py

```diff
@@ -0,0 +1,10 @@
+#Python Script To Check Number Is Prime Or Not
+def check_primeno(no):
+    for i in range(2,no):
+        if no%i==0:
+            print(f'{no} Is Not Prime Number')

```

Comment:
what is 'f' inside the print.

is there any other way to write this statement?

Reply:
f inside the print is f-string.
here i use f-string  because if i print message along with variable without using f_string or .format( ) , interpreter will provide error.
 because  it does not allow display message along with value which variable can hold, in short , if i print message and variable into quotation mark then error will be occurred so overcome this problem i use f-string

Yes, there is different way to print this statement 

1st method is : using comma operator
For example : print(no ,"Is Not Prime Number") 
i did not use this method because python automatically converts this variable into string while using comma operator.

2nd method is : using .format( ) method
For example : print("{} is even number".format(no))